internal-dependencies/pom.xml

```diff
@@ -116,6 +116,14 @@
         <version>1.7.1</version>
         <scope>provided</scope>
       </dependency>
+
+    <dependency>
+      <groupId>commons-collections</groupId>
+      <artifactId>commons-collections</artifactId>
```

Comment:
❓ We have never pulled this in before, as far as I know. Do we need to check the license? Do we only use this in test classes or do we ship this now as well? 🤔

Reply:
I see, thanks for the explanation 👍